### PR TITLE
Add note to access to GitHub using sigdev

### LINKED ISSUE #1425: Add deployment key for the sigdev user documentation

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1425

### CODE REVIEW

File: doc/integrator/deploy.rst [hunk 1]
```diff
@@ -27,6 +27,9 @@ Located in ``/var/www/vhost/<project_vhost>/private/<project>``.
 
 Only user ``sigdev`` may access this environment.  Developers must then prefix
 all the commands on this environment by ``sudo -u sigdev``.
+
+To be able to access to github you should use the https protocol.
```
Comment (elemoine) [inline doc/integrator/deploy.rst]: github -> GtiHub :)

File: doc/integrator/deploy.rst [hunk 2]
```diff
@@ -34,6 +37,13 @@ For instance to update the application:
     sudo -u sigdev make -f demo.mk update
     sudo -u sigdev make -f demo.mk build
 
+.. note::
+
+   If you need to access to github by using the ssh protocole you
```
Comment (elemoine) [inline doc/integrator/deploy.rst]: protocole -> prototcol

File: doc/integrator/deploy.rst [hunk 3]
```diff
@@ -34,6 +37,13 @@ For instance to update the application:
...
+   should add the sigdev key locate in ``/var/sig/.ssh/id_rsa.pub``
```
Comment (elemoine) [inline doc/integrator/deploy.rst]: locate -> located

File: doc/integrator/deploy.rst (hunk 1 above)
Comment (fredj) [inline doc/integrator/deploy.rst]: GtiHub -> GitHub :-)

File: doc/integrator/deploy.rst (hunk 2 above)
Comment (kalbermattenm) [inline doc/integrator/deploy.rst]: prototcol -> protocol :-)

File: doc/integrator/deploy.rst (hunk 1 above)
Comment (elemoine) [inline doc/integrator/deploy.rst]: Thansk :)

**Comment by sbrunner** [comment]
Thanks to all, updated :-)

File: doc/integrator/deploy.rst (hunk 2 above)
Comment (elemoine) [inline doc/integrator/deploy.rst]: Tahnsk!

**Pull request merged**